railgun/test/integration/suite_test.go

```diff
@@ -98,19 +125,25 @@ func deployControllers(ctx context.Context, ready chan ktfkind.ProxyReadinessEve
 		}
 		kubeconfig.Close()
 
+		// deploy our CRDs to the cluster
+		cmd := exec.CommandContext(ctx, "kubectl", "--kubeconfig", kubeconfig.Name(), "apply", "-f", "../../config/crd/bases/configuration.konghq.com_udpingresses.yaml")
```

Comment:
What makes it necessary to apply the UDPIngress CRD this way? We don't have anything similar for the other CRDs, so I'm curious why this one is the odd one out. Is

Reply:
It was at the time the quickest simplest thing which met the need, but as indicate this entire function will need to be reworked for KIC 2.0.

I do have some code in my git stash right now that reworks this and runs the controller as a goroutine, but I didn't want to add it to this scope and overload it, https://github.com/Kong/kubernetes-ingress-controller/issues/1128 is the eventual pre-2.0 follow up which handles the scope of this.